packages/veritone-widgets/src/widgets/Table/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { func, string, number, arrayOf, bool, shape, object } from 'prop-types';
+import { Table, PaginatedTable, Column, MenuColumn } from 'veritone-react-common';
+import { omit } from 'lodash';
+import widget from '../../shared/widget';
+
+
+class TableWidget extends React.Component {
+  static propTypes = {
+    data: arrayOf(object).isRequired,
+    columns: arrayOf(shape({
+      dataKey: string.isRequired,
+      header: string,
+      transform: func,
+      menu: bool,
+      onSelectItem: func,
+      cursorPointer: bool,
+      align: string,
+      width: number
+    })),
+    paginate: bool,
+    initialItemsPerPage: number,
+    onCellClick: func,
+    focusedRow: number,
+    renderFocusedRowDetails: func,
+    onShowCellRange: func,
+    onRefreshPageData: func,
+    emptyMessage: string
+  };
+  
+  getRowData = (i) => {
+    return this.props.data[i];
+  }
+
+  render() {
+    const TableComp = this.props.paginate ? PaginatedTable : Table;
+    const tableProps = omit(this.props, ['data', 'columns', 'paginate']);
+    const tableColumns = this.props.columns.map((column, idx) => {
+      if (column.menu) {
+        return (
+          <MenuColumn
+            key={idx}
```

Comment:
this should be datakey probably, don't use index as a key (justification [here](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). basically a changing index doesn't capture the correct information for a key because data order can change.)